interface/patient_file/encounter/forms.php

```diff
@@ -589,19 +589,22 @@ function myGetRegistered($state = "1", $limit = "unlimited", $offset = "0")
                 $new_category_ = $new_category;
                 $new_category_ = str_replace(' ', '_', $new_category_);
                 if ($old_category != '') {
-                    $StringEcho .= "</table></div></li>";
+                    $StringEcho .= "</div></div>";
                 }
-                $StringEcho .= "<li class=\"encounter-form-category-li\"><a href='JavaScript:void(0);' onClick=\"mopen(" . attr_js($DivId) . ");\" >" . text($new_category) . "</a><div id='" . attr($DivId) . "' ><table border='0' cellspacing='0' cellpadding='0'>";
+                $StringEcho .= "<div class='dropdown d-inline'><button class='btn btn-secondary dropdown-toggle' type='button' id='menu" . text($new_category) . "' data-toggle='dropdown' aria-haspopup='true' aria-expanded='false'>" .text($new_category) . "</button>";
```

Comment:
$new_category should be wrapped in the attr() function since it's within html attribute. The text/attr are html escaping functions and are basically shortcuts for htmlspecialchar/ENT_NOQUOTES and htmlspecialchars/ENT_QUOTES respectively.